src/control/controlcompressingproxy.cpp

```diff
@@ -0,0 +1,24 @@
+#include "control/controlcompressingproxy.h"
+
+#include "moc_controlcompressingproxy.cpp"
+
+// Compressor
+CompressingProxy::CompressingProxy(QObject* parent)
+        : QObject(parent){};
+
+bool CompressingProxy::emitCheck() {
```

Comment:
This function does not *emit* any signals and the naming is misleading.

Reply:
I renamed it to isLatestEventInQueue and added some lines of text, that explain what this function does. Hopefully it's better understandable now.